src/test/java/uk/gov/pay/api/it/PaymentsResourceGetIT.java

```diff
@@ -765,6 +778,21 @@ public void getPaymentWithNoAuthorisationSummaryThroughConnector() {
         response.body("authorisation_summary", is(nullValue()));
     }
 
+    @Test
+    public void getPaymentShouldNotIncludeNoAuthorisationSummaryThroughConnectorWhenThreeDSecureRequiredIsFalse() {
+        AuthorisationSummary authorisationSummary = new AuthorisationSummary(new ThreeDSecure(false));
+        connectorMockClient.respondWithChargeFound(CHARGE_TOKEN_ID, GATEWAY_ACCOUNT_ID,
+                getConnectorCharge()
+                        .withAuthorisationSummary(authorisationSummary)
+                        .build());
+
+        ValidatableResponse response = getPaymentResponse(CHARGE_ID);
+
+        assertCommonPaymentFields(response);
+        assertConnectorOnlyPaymentFields(response);
+        response.body("authorisation_summary", is(nullValue()));
+    }
+
```

Comment:
Am I right in thinking existing tests cover there being no `AuthorisationSummary` at all (essentially that the null checks are in place)?

Reply:
Good question, I'll check

Reply:
Yes, I added it in 2021 `getPaymentWithNoAuthorisationSummaryThroughConnector`

Reply:
And now added `getPaymentWithNoAuthorisationSummaryThroughLedger`